lib/archethic/self_repair.ex

```diff
@@ -83,4 +91,63 @@ defmodule Archethic.SelfRepair do
     |> CronScheduler.get_previous_run_date!(DateTime.to_naive(date_from))
     |> DateTime.from_naive!("Etc/UTC")
   end
+
+  @spec start_notifier(list(Node.t()), list(Node.t()), DateTime.t()) :: :ok
+  def start_notifier(prev_available_nodes, new_available_nodes, availability_update) do
+    diff_node =
+      (prev_available_nodes -- new_available_nodes)
+      |> Enum.reject(&(&1.first_public_key == Crypto.first_node_public_key()))
+
+    case diff_node do
+      [] ->
+        :ok
+
+      nodes ->
+        unavailable_nodes = Enum.map(nodes, & &1.first_public_key)
+
+        DynamicSupervisor.start_child(
+          NotifierSupervisor,
+          {Impl,
```

Comment:
I think you mean `Notifier` here.
Also it would be good to give some doc for the function.